### DISCUSSION TITLE
June Personal Web - Include Dotenv File to Makefile

### DISCUSSION THREAD

**Discussion by giscus** [ClavinJune/ClavinJune.github.io#16] [General]
# June Personal Web - Include Dotenv File to Makefile

sunday snippet #17 include dotenv file to makefile

https://clavinjune.dev/en/blogs/include-dotenv-file-to-makefile/

**Comment by jenciso** [comment]
I use this one:
```
include .env
export
```
So, `export` command permit pass the variable to container

**Reply by clavinjune** [reply]
yes @jenciso, `include` directive only include variables from `.env`. it is not necessarily export it as an environment variable.

use `export` If you want to pass the variables to another script as variable
